cicd/deploy_ephemeral_db.sh

```diff
@@ -10,37 +12,41 @@ DB_DEPLOYMENT_NAME="${DB_DEPLOYMENT_NAME:-$COMPONENT_NAME-db}"
 # Deploy k8s resources for app without its dependencies
 export BONFIRE_NS_REQUESTER="${JOB_NAME}-${BUILD_NUMBER}-db"
 NAMESPACE=$(bonfire namespace reserve)
-DB_NAMESPACE=$NAMESPACE  # track which namespace was used here for 'teardown' in common_deploy_logic
+export DB_NAMESPACE="$NAMESPACE"  # track which namespace was used here for 'teardown' in common_deploy_logic
 # TODO: add code to bonfire to deploy an app if it is defined in 'sharedAppDbName' on the ClowdApp
 # TODO: add a bonfire command to deploy just an app's DB
 set -x
 bonfire process \
-    $APP_NAME \
+    "$APP_NAME" \
     --source=appsre \
-    --ref-env ${REF_ENV} \
-    --set-template-ref ${COMPONENT_NAME}=${GIT_COMMIT} \
-    --set-image-tag $IMAGE=$IMAGE_TAG \
-    --namespace $NAMESPACE \
+    --ref-env "$REF_ENV" \
+    --set-template-ref "${COMPONENT_NAME}=${GIT_COMMIT}" \
+    --set-image-tag "${IMAGE}=${IMAGE_TAG}" \
+    --namespace "$NAMESPACE" \
     --optional-deps-method none \
-    $COMPONENTS_ARG \
-    $COMPONENTS_RESOURCES_ARG | oc_wrapper apply -f - -n $NAMESPACE
+    "$COMPONENTS_ARG" \
+    "$COMPONENTS_RESOURCES_ARG" | oc_wrapper apply -f - -n "$NAMESPACE"
 
-bonfire namespace wait-on-resources $NAMESPACE --db-only
+bonfire namespace wait-on-resources "$NAMESPACE" --db-only
 
 # Set up port-forward for DB
 LOCAL_DB_PORT=$(python -c 'import socket; s=socket.socket(); s.bind(("", 0)); print(s.getsockname()[1]); s.close()')
-oc_wrapper port-forward svc/$DB_DEPLOYMENT_NAME $LOCAL_DB_PORT:5432 -n $NAMESPACE &
-PORT_FORWARD_PID=$!
```

Comment:
is this okay to go @bsquizz  ? I don't see it used anywhere .

Reply:
I am surprised but yeah, I see no reference to the stored pid anywhere so I think you are right -- it can go.